test/java/seedu/addressbook/commands/FindCommandTest.java

```diff
@@ -0,0 +1,97 @@
+package seedu.addressbook.commands;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import seedu.addressbook.data.AddressBook;
+import seedu.addressbook.data.exception.IllegalValueException;
+import seedu.addressbook.data.person.Person;
+import seedu.addressbook.data.person.ReadOnlyPerson;
+import seedu.addressbook.util.TestUtil;
+import seedu.addressbook.util.TypicalPersons;
+
+/**
+ * Test class for the Find Command
+ */
+public class FindCommandTest {
+
+    private AddressBook addressBook;
+    private AddressBook unmutatedAddressBook;
+
+    @Before public void setUp() {
+        this.addressBook = new TypicalPersons().getTypicalAddressBook();
+        this.unmutatedAddressBook = new TypicalPersons().getTypicalAddressBook();
+    }
+
+    @Test
+    public void execute_sameWordSameCase_matched() throws IllegalValueException {
+        List<ReadOnlyPerson> expected = TestUtil.createList(new Person(TypicalPersons.amy));
+        
+        String[] keywordsWithMatchingCase = {"Amy"};
+        
+        FindCommand command = createFindCommand(keywordsWithMatchingCase);
+        assertCommandBehavior(command, Command.getMessageForPersonListShownSummary(expected),
+                                                                    unmutatedAddressBook, addressBook);
+    }
+    
+    @Test
+    public void execute_sameWordDifferentCase_notMatched() throws IllegalValueException {
+        List<ReadOnlyPerson> expected = Collections.emptyList();
```

Comment:
Alternatively, you can create this utility method:
```java
public void assertNoResults(...){
    assertCommandBehavior(..., Collections.emptyList(), ...);
}
```
In that case you may want to rename `assertCommandBehavior` as `assertResults` (for symmetry)